code/embeddings.py

```
@@ -0,0 +1,26 @@
+import pandas as pd
+from sentence_transformers import SentenceTransformer
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)